bin/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
```

Comment:
I think this file can be deleted
- `node_modules` should be listed in the `.gitignore` in the root dir of this repo
- `npm-debug.log` also
- `yarn.lock` should be committed